### PR TITLE
Fix use of timeout in pushAppBundle

### PR DESCRIPTION
**Pull request by Volune**
Fix timeout error when installing new app (wrong order of arguments for `.timeout()`)

Probably related to appium/appium#13180 and appium/appium#13218

### CODE REVIEW

File: lib/ios-deploy.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ class IOSDeploy {
             writeStream.on('error', reject);
           });
           readStream.pipe(writeStream);
-          await itemPushWait.timeout(`Couldn't push '${itemPath}' within the timeout ${ITEM_PUSH_TIMEOUT}ms`, ITEM_PUSH_TIMEOUT);
+          await itemPushWait.timeout(ITEM_PUSH_TIMEOUT, `Couldn't push '${itemPath}' within the timeout ${ITEM_PUSH_TIMEOUT}ms`);
```
Comment (mykola-mokhnach) [inline lib/ios-deploy.js]: actually, the second argument should be an error instance. Please also wrap the string into `new Error()`

**Comment by mykola-mokhnach** [comment]
Thanks @Volune , this was a silly mistake :(

**Comment by KazuCocoa** [comment]
great 🎉

**Comment by jovana119** [comment]
Hi, which version should I install to get this changes? appium@beta, appium@1.15.0-rc.1 or sth else?

**Comment by KazuCocoa** [comment]
@jovana119 It will be available as `@beta` soon (I bumped the version up as 2.131.1)

**Pull request merged**